core/src/finding/find_tracks.hpp

```diff
@@ -76,6 +76,8 @@ track_candidate_container_types::host find_tracks(
     using propagator_type =
         detray::propagator<stepper_t, navigator_t, actor_type>;
 
+    assert(config.min_track_candidates_per_track > 1);
```

Comment:
Why not `>=`? I think 1 should be allowed

Reply:
Good catch, changed to `>= 1`